src/MOI_wrapper.jl

```diff
@@ -219,10 +219,17 @@ function MOI.supports_add_constrained_variables(
     return MOI.supports_add_constrained_variables(model.optimizer, MOI.Reals)
 end
 
+function _assert_parameter_is_not_nan(set::MOI.Parameter{T}) where {T}
+    if isnan(set.value)
```

Comment:
Lets switch to "isfinite" that checks both NaN and Inf?

Reply:
I was wondering if Inf was a valid parameter. But ok I will change it

Reply:
I was thinking the same, but since JuMP blocks this, we should block as well

Reply:
is someone shows up with a use case, we can review